style.css

```diff
@@ -1,30 +1,40 @@
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: auto;
+}
 p {
-  display: inline;
+  margin: 0;
   font-size: 50px;
   font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
```

Comment:
This should be in general selector at the top of the file -> `* { font-family ... }`